allennlp/modules/seq2vec_encoders/boe_encoder.py

```diff
@@ -0,0 +1,62 @@
+from overrides import overrides
+
+import torch
+
+from allennlp.common import Params
+from allennlp.modules.seq2vec_encoders.seq2vec_encoder import Seq2VecEncoder
+from allennlp.nn.util import get_lengths_from_binary_sequence_mask
+
+SMALL = 1e-5
+
+@Seq2VecEncoder.register("boe")
+class BagOfEmbeddingsEncoder(Seq2VecEncoder):
+    """
+    A ``BagOfEmbeddingsEncoder`` is a simple :class:`Seq2VecEncoder` which simply sums the embeddings
+    of a sequence across the time dimension. The input to this module is of shape ``(batch_size, num_tokens,
+    input_dim)``, and the output is of shape ``(batch_size, output_dim)``.
```

Comment:
I would put `embedding_dim` here rather than "input_dim" and "output_dim", since that's what it's called elsewhere (and it makes clearer that they're the same)